azerothcore/azerothcore-wotlk
# fix(DB/Creature): Engineer "Spark" and Geezle

## Changes Proposed:
- Add Engineer "Spark" Overgrind the correct dialogs
- Fix Engineer's "Spark" combat bugs
- Move all the scripts for the quests "Tree's Company" and "Show Gnomercy" from C++ to SAI
- Add correct texts to Enginner "Spark" with the corect emotes.
- Add correct emotes to Geezle


## Issues Addressed:
- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/4086



## Tests Performed:
- Windows



## How to Test the Changes:

First fix:
- .go c 61966
- .q a 9537
- Speak with Engineer "Spark"
- Test two things: Aggro the Engineer and then get out of combat and see if he resets to the normal state, the second time kill him.
Second fix:
- .go xyz -5086.585 -11248.529 0.621
- .q a 9531
- Use the item
- Watch the script and make sure you get the quest completed.

## Known Issues and TODO List:



## Target Branch(es):








 
## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR
